src/ECharts/Theme.js

```diff
@@ -1,8 +1,12 @@
-require("echarts/theme/dark")
-require("echarts/theme/infographic")
-require("echarts/theme/macarons")
-require("echarts/theme/roma")
-require("echarts/theme/shine")
-require("echarts/theme/vintage")
+"use strict";
 
-exports.forceExport = null
+exports._dark = { name: "dark", value: require("echarts/theme/dark") };
```

Comment:
I think it shouldn't as in examples we use exported values
https://github.com/slamdata/purescript-echarts/blob/5066e5e957b83693fc08cce264f0a0729078b050/example/src/Scatter.purs#L98-L99